hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMRatisServerImpl.java

```diff
@@ -229,10 +229,19 @@ public void stop() throws IOException {
   public List<String> getRatisRoles() throws IOException {
     Collection<RaftPeer> peers = division.getGroup().getPeers();
     List<String> ratisRoles = new ArrayList<>();
+    InetAddress peerInetAddress = null;
     for (RaftPeer peer : peers) {
-      InetAddress peerInetAddress = InetAddress.getByName(
-              HddsUtils.getHostName(peer.getAddress()).get());
-      boolean isLocal = NetUtils.isLocalAddress(peerInetAddress);
+      try {
```

Comment:
NIT: narrow down the lifecycle of `peerInetAddress`
```
for (RaftPeer peer : peers) {
  InetAddress peerInetAddress = null;
  try {
    //
  }
```

Reply:
Done.